packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -59,6 +59,7 @@ async function handler(req: NextApiRequest & { userId?: number }) {
           type: true,
           externalCalendarId: true,
           credentialId: true,
+          credential: true,
```

Comment:
Since we longer have one user per booking we can't rely on getting credentials that way. We can however nest credentials in the booking reference record, that way we can automatically find related credentials and opens us up to update multiple calendars with a single booking.